docs/deployment/docker-deployment.md

```diff
@@ -52,7 +52,13 @@ If postgres connection errors occur, try your ip for POSTGRES_HOST. For all Envi
 | POSTGRES_DB*   |         | Database name in Postgres                                     |
 | POSTGRES_USER* |         | Authorized username for database                              |
 | POSTGRES_PASS* |         | Password of authorized user                                   |
-| HTTP_PROXY     |         | Proxy server and port for outbound HTTP requests, e.g. "proxy.example.de:8080" |
+| HTTP_PROXY_URL |         | Proxy server and port for outbound HTTP requests, e.g. "proxy.example.de:8080" |
```

Comment:
Blaze [uses](https://github.com/samply/blaze/blob/develop/docs/deployment/environment-variables.md#other-environment-variables) PROXY_HOST, PROXY_PORT, PROXY_USER and PROXY_PASSWORD. Do we really need the differentiation between HTTP and HTTPS? It would be good if all Samply projects use the same env var names for the same things.

Reply:
@djuarezgf told me, that some sites really need the differentiation. There are even sites, that have different login information for http and https proxy.